README.md

```diff
@@ -59,6 +61,56 @@ Some of the controls available include:
+- [FloatingActionButton](fluentui_controls/src/main/java/com/microsoft/fluentui/tokenized/controls/FloatingActionButton.kt)
+- [Label](fluentui_controls/src/main/java/com/microsoft/fluentui/tokenized/controls/Label.kt)
+- [LinearProgressIndicator](fluentui_progress/src/main/java/com/microsoft/fluentui/tokenized/progress/LinearProgressIndicator.kt)
+- [ListContentBuilder](fluentui_listitem/src/main/java/com/microsoft/fluentui/tokenized/contentBuilder/ListContentBuilder.kt)
+- [ListItem](fluentui_listitem/src/main/java/com/microsoft/fluentui/tokenized/listitem/ListItem.kt)
+- [Menu](fluentui_menus/src/main/java/com/microsoft/fluentui/tokenized/menu/Menu.kt)
+- [NotificationCommon](fluentui_notification/src/main/java/com/microsoft/fluentui/tokenized/notification/NotificationCommon.kt)
+- [PeoplePicker](fluentui_peoplepicker/src/main/java/com/microsoft/fluentui/tokenized/peoplepicker/PeoplePicker.kt)
+- [Persona](fluentui_persona/src/main/java/com/microsoft/fluentui/tokenized/persona/Persona.kt)
+- [PersonaChip](fluentui_persona/src/main/java/com/microsoft/fluentui/tokenized/persona/PersonaChip.kt)
+- [PersonaList](fluentui_persona/src/main/java/com/microsoft/fluentui/tokenized/persona/PersonaList.kt)
+- [Pill](fluentui_tablayout/src/main/java/com/microsoft/fluentui/tokenized/segmentedcontrols/Pill.kt)
+- [PillSwitch](fluentui_tablayout/src/main/java/com/microsoft/fluentui/tokenized/segmentedcontrols/PillSwitch.kt)
+- [PillTabs](fluentui_tablayout/src/main/java/com/microsoft/fluentui/tokenized/segmentedcontrols/PillTabs.kt)
+- [ProgressText](fluentui_progress/src/main/java/com/microsoft/fluentui/tokenized/progress/ProgressText.kt)
+- [RadioButton](fluentui_controls/src/main/java/com/microsoft/fluentui/tokenized/controls/RadioButton.kt)
+- [SearchBar](fluentui_topappbars/src/main/java/com/microsoft/fluentui/tokenized/SearchBar.kt)
+- [SearchBarPersonaChip](fluentui_persona/src/main/java/com/microsoft/fluentui/tokenized/persona/SearchBarPersonaChip.kt)
+- [Shimmer](fluentui_progress/src/main/java/com/microsoft/fluentui/tokenized/shimmer/Shimmer.kt)
+- [SideRail](fluentui_tablayout/src/main/java/com/microsoft/fluentui/tokenized/navigation/SideRail.kt)
+- [Snackbar](fluentui_notification/src/main/java/com/microsoft/fluentui/tokenized/notification/Snackbar.kt)
+- [TabBar](fluentui_tablayout/src/main/java/com/microsoft/fluentui/tokenized/navigation/TabBar.kt)
+- [TabItem](fluentui_listitem/src/main/java/com/microsoft/fluentui/tokenized/tabItem/TabItem.kt)
+- [TextField](fluentui_controls/src/main/java/com/microsoft/fluentui/tokenized/controls/TextField.kt)
+- [ToggleSwitch](fluentui_controls/src/main/java/com/microsoft/fluentui/tokenized/controls/ToggleSwitch.kt)
+- [ToolTip](fluentui_notification/src/main/java/com/microsoft/fluentui/tokenized/notification/ToolTip.kt)
+- [Utils](fluentui_persona/src/main/java/com/microsoft/fluentui/tokenized/persona/Utils.kt)
```

Comment:
This can be removed as it is not a comtrol

Reply:
Done